app/views/notification_mailer/meeting_reminder.text.erb

```diff
@@ -0,0 +1,4 @@
+Hi <%= @member.name %>,
+
+Here's a friendly reminder that your meeting tomorrow, "<%= @meeting.name%>", will be taking place at <%= @meeting.time %> at 
```

Comment:
Good question. @sienatime is working on the i18n project now. I'm thinking we have a language picker that saves the users preferences to local storage? Ideas/thoughts?